.github/workflows/build.yml

```diff
@@ -0,0 +1,47 @@
+name: Build Demos
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-python@v4
+        with:
+          python-version: '3.9'
+      - name: Install General Dependencies
+        run: sudo apt-get install -y make build-essential subversion libsdl2-dev
+      - name: Install Python Dependencies
+        run: pip install -r tools/requirements.txt
+      - name: Install cc65
+        run: |
+          cd ..
+          git clone https://github.com/cc65/cc65.git
+          cd cc65
+          make -j
+          sudo make install
+      - name: Install ACME
+        run: |
+          cd ..
+          svn checkout svn://svn.code.sf.net/p/acme-crossass/code-0/trunk acme-crossass-code-0
+          cd acme-crossass-code-0/src
+          make -j
+          sudo make install
+      - name: Download Latest X16-Emu
+        run: |
+          cd ..
+          gh run download -R commanderx16/x16-emulator -n "X16-Emu Linux 64-bit" --dir x16-emulator
+          cd x16-emulator
+          tar --strip-components=1 -xf x16emu.tar.gz
+          ./x16emu -version
```

Comment:
I'm not sure I understand the purpose of running x16emu as part of the build CI, especially just to dump the version information. x16emu has its own CI, if it needs to be tested for execution that should be done as part of that build process.

Reply:
Part of the build process defined in this repo's build file is the conversion of `.BAS` to `.PRG` files, done in the `bas2prg.py` file, and that program invokes the x16-emu with `x16emu -bas <basic file>` (plus a small additional to dump the tokenized basic to a PRG file in the end. Hence to `make` the demos, a executable x16-emu is needed.